App.js

```diff
@@ -47,6 +48,7 @@ const HeaderSimple = ({name}) =>{
 const App = () => {
   return (
     <DriverState>
+      <TractorState>
```

Comment:
What if we want to access Driver inside tractor and vice versa?

Reply:
Then,May be we can try to use one maincontext which contain all context as it value.I don't know much about it but we try Anna.